h/session.py

```diff
@@ -110,7 +111,22 @@ def includeme(config):  # pragma: no cover
         httponly=True,
         secure=secure,
         serializer=JSONSerializer(),
-        timeout=3600,
```

Comment:
The previous one-hour lifetime of sessions seems to originate from here: https://github.com/hypothesis/h/pull/3900#discussion_r79880001